pkgs/data/themes/klassy/default.nix

```
@@ -0,0 +1,42 @@
+{mkDerivation
+, lib
+, kdecoration
+, qt5
+, cmake
+, extra-cmake-modules
+, plasma-workspace
+, qtbase
+, fetchFromGitHub
+}:
+
+mkDerivation rec {
+  pname = "klassy";
+  version = "4.1";
+
+  src = fetchFromGitHub {
+    owner = "paulmcauley";
+    repo = "klassy";
+    rev = "972edd08184b3a416b166053a1b1a3d042d33d92";
```

```suggestion
  version = "4.1.breeze5.25.80";

  src = fetchFromGitHub {
    owner = "paulmcauley";
    repo = "klassy";
    rev = version;
```
thats the actual version according to cmake